examples/robot_localization_failure/recordings/.gitignore

```diff
@@ -2,3 +2,5 @@
 /rec_20250618_113817
 /rec_20250618_134101
 /rec_20250704_173434
+/rec_20250704_173434_sliced
+/rec_20250707_114940
```

Comment:
what is about this recording? It was added to the gitignore, but there is no respective dvc file?